sdk/keyvault/azure-keyvault-secrets/tests/test_secrets_client.py

```diff
@@ -350,3 +350,28 @@ class _CustomHookPolicy(object):
     @KeyVaultClientPreparer(client_kwargs={"custom_hook_policy": _CustomHookPolicy()})
     def test_custom_hook_policy(self, client, **kwargs):
         assert isinstance(client._client._config.custom_hook_policy, SecretClientTests._CustomHookPolicy)
+
+    @ResourceGroupPreparer(random_name_enabled=True)
+    @KeyVaultPreparer()
+    @KeyVaultClientPreparer()
+    def test_continuation_token(self, client, **kwargs):
+        secret_name = "crud-secret"
+        secret_value = self.get_resource_name("crud_secret_value")
+
+        # create secret
+        secret = client.set_secret(secret_name, secret_value)
+
+        # delete secret
+        initial_poller = client.begin_delete_secret(secret.name)
+        continuation_token = initial_poller.continuation_token()
+        poller = client.begin_delete_secret(secret.name, continuation_token=continuation_token)
+        deleted_secret = poller.result()
+        self.assertIsNotNone(deleted_secret)
```

Comment:
`deleted_secret` could be some arbitrary incorrect value though. We want to verify the poller was correctly rehydrated; does this work?

```suggestion
        self._assert_secret_attributes_equal(initial_poller.result, poller.result)
```

Reply:
great idea, added a check here. thanks!